scripts/post-deploy-configure-k8s.sh

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+LB_CA_CERT="$1"
+SYSTEM_DOMAIN="$2"
+
+BOSH_ENV_NAME="$(bosh env --json | jq .Tables[0].Rows[0].name -r)"
+DOPPLER_ADDRESS="$(bosh -d cf vms | grep doppler | cut -d$'\t' -f4 | head -n 1)"
+
+echo "Creating secret for the load balancer CA cert..."
+kubectl apply -f <(kubectl create secret generic lb-ca-cert \
+  --from-literal=ca.crt="$LB_CA_CERT" \
+  --dry-run \
+  --save-config \
+  -o yaml)
+kubectl apply -f <(cat <<EOD
+---
+apiVersion: apps/v1
```

Comment:
@jenspinney what do you think about having these as templates, and create a job that we can all use as an errand?